src/config.ts

```diff
@@ -37,7 +57,7 @@ export interface StampsConfigAutobuy {
   refreshPeriod: number
 }
 
-export type StampsConfig = StampsConfigHardcoded | StampsConfigAutobuy | StampsConfigExtends
+export type StampsConfig = StampsConfigHardcoded | StampsConfigAutobuy | StampsConfigExtends | StampsConfigReupload
```

Comment:
Again, this is not another "stamp management mode" but rather standalone feature, so should not be part of `StampsConfig`